nexus/templates/nexus/base.html

```diff
@@ -13,6 +13,15 @@
 
         <meta name="robots" content="NONE,NOARCHIVE">
 
+        <script>
```

Comment:
this is no longer the recommended way to deploy JavaScript on the django admin. See https://docs.djangoproject.com/en/dev/ref/contrib/admin/javascript/